Casks/font/font-d/font-darwin.rb

```diff
@@ -0,0 +1,15 @@
+# typed: true
+# frozen_string_literal: true
+
+cask "font-darwin" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/topological-modular-forms/Darwin-Typeface/raw/main/output/DarwinSerif-Regular.ttf"
```

Comment:
```suggestion
  url "https://github.com/topological-modular-forms/Darwin-Typeface/raw/main/output/DarwinSerif-Regular.otf"
```
OTF is preferred over TTF